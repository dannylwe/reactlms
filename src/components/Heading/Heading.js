import React, { Component } from 'react'

class Heading extends Component {
  render() {
    return (
      <div>
        <h1>This is a Heading</h1>
      </div>
    )
  }
}

export default Heading

