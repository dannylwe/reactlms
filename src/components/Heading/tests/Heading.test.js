import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Heading from '../Heading';

describe('Heading', () => {
  it('should render without failing', () => {
    const wrapper = renderer.create(<MemoryRouter><Heading /></MemoryRouter>);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
