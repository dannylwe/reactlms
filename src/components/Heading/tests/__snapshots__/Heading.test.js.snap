// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading should render without failing 1`] = `
<div>
  <h1>
    This is a Heading
  </h1>
</div>
`;
